src/tds/unittests/convert.c

```diff
@@ -242,7 +242,7 @@ main(int argc, char **argv)
 			srclen = sizeof(tds_uint8);
 			break;
 		case SYBFLT8:
-			tds_float = 3.14159;
+			tds_float = 0.000001;
```

Comment:
not sure if would be better for this case to extend t0007 test instead. I didn't manage to reproduce the error under Linux. Which environment are you using?

Reply:
Looks fine, nice work - reflects the problem as well.